lib/periodic_jobs.rb

```diff
@@ -43,6 +43,8 @@
   # Update Lighthouse526DocumentUpload statuses according to Lighthouse Benefits Documents service tracking
   mgr.register('0 3 * * *', 'Form526StatusPollingJob')
   # Updates status of FormSubmissions per call to Lighthouse Benefits Intake API
+  mgr.register('0 2 * * 0', 'Form526StatusPollingJob', 'args' => { paranoid: true })
+  # Checks all 'success' type submissions in LH to ensure they haven't changed
```

Comment:
@SamStuckey A few things:
1. Why don't we have specs for this file, as described in [the docs](https://github.com/sidekiq/sidekiq/wiki/Ent-Periodic-Jobs#testing)? 😕 I'm not saying write all the specs for this file, but did you try to write up a little test for this?
2. Is `=>` right? I don't see that syntax in the docs (linked above). I see the `:` syntax. Oh wait. Maybe [I'm wrong](https://github.com/sidekiq/sidekiq/wiki/Ent-Periodic-Jobs#notes)
3. why in the world do we comment _below_ each line in this file?? 😆 😂

Reply:
1. good call I'll add a spec
2. `:` is my preference, but hash rocket is what they have in the docs and should do the same thing so ¯\_(ツ)_/¯ 
3. yeah that annoyed me too... i thought about moving them all but that would've made a much bigger PR

Reply:
Ok, figured  out why there's no spec 😓 it's because configuring it is weird... i'm having trouble getting the sidekiq jobs to be available in the spec file. the  spec i tried looks like this

```
require 'rails_helper'

Dir[Rails.root.join('app/sidekiq/**/*')].sort.each { |file| require file }

RSpec.describe 'Periodic Jobs', type: :request do
  it 'verifies that periodic jobs have been defined with correct syntax' do
    require 'periodic_jobs'
    require 'sidekiq-ent/periodic/testing'
    ct = Sidekiq::Periodic::ConfigTester.new
    ct.verify(&PERIODIC_JOBS)
  end
end
```

but i'm getting  'cannot load ...' type errors.

Reply:
Can we ticket this for later work? feels out of scope to me, but I defer to you

Reply:
totally.